src/screens/MemberDetail/MemberDetail.tsx

```
@@ -464,16 +560,131 @@
+                  className={`${styles.inputColor} small form-control d-flex align-items-center`}
+                  style={{ width: 'fit-content' , gap: "0.5rem", border:'solid', borderWidth:'2px', borderColor:'#ced4da' }}
+                >
+                  <GlobalIcon />
+                  <span>English</span>
+                </button>
+              </Col>
+
+              <Col md={7}>
+                <label className="form-label" style={{color : '#495057'}} >
+                  {t('deleteUser')}
+                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)